install_config/cluster_metrics.adoc

```diff
@@ -570,6 +570,11 @@ information can be found under the
 link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift
 Agent documentation].
 
+The agent runs as a daemon set in your cluster and will be deployed to the
```

Comment:
s/will be deployed/is deployed
s/will continue to be able to monitor/continues to monitor
s/plugin/plug-in